Lesson16/bilge/script.js

```diff
@@ -0,0 +1,305 @@
+// 30.0 and Above	Obesity
+
+function findBodyMassIndex (weight, height) {
+  let bmi = weight / (height ** 2); 
+
+  if (bmi < 18.5) {
+    return 'The person is underweight.'
+  }
+  else if (bmi >= 18.5 && bmi <= 24.9) {
+    return 'The person is normal.'
+  }
+  else if (bmi >=25 && bmi <= 29.9) {
+    return 'The person is overweight.'
+  }
+  else {
+    return 'The person is obese.'
+  }
+}
+
+console.log('Ex. 8:');
+
+console.log(findBodyMassIndex(40, 1.70));
+console.log(findBodyMassIndex(60, 1.70));
+console.log(findBodyMassIndex(80, 1.70));
+console.log(findBodyMassIndex(200, 1.70));
+
+// 9. Write a function that takes in a person's age and checks if they are a child (age 0-12), teenager (age 13-19), adult (age 20-64), or senior citizen (age 65 and above) using if else statement.
+
+function checkAge(age) {
+	if (age >= 0 && age <= 12) {
+		return 'You are a child.'
+  }
+	else if (age >= 13 && age <= 19) {
+		return 'You are a teenager.'
+  }
+	else if (age >= 20 && age <= 64) {
+		return 'You are an adult.'
+  }
+  else {
+    return 'You are a senior citizen.'
+  }
+}
+
+console.log('Ex. 9:');
+
+console.log(checkAge(12));
+console.log(checkAge(19));
+console.log(checkAge(64));
+console.log(checkAge(65));
+
+// 10. Write a function that takes age as a number and whether they are a student (yes or no). If the age is less than 18 and the user is a student, print "You are a student and not old enough to vote". If the age is between 18 and 65 (inclusive) and the user is not a student, print "You are eligible to vote". If the age is greater than 65, print "You are a senior citizen". If the user is a student but their age is greater than or equal to 18, print "You are old enough to vote".
+
+function checkVoteEligibility (age, student) {
+	if (age < 18 && student) {
+		return 'You are a student and not old enough to vote.'
+  }
+	else if (age >= 18 && student) {
+		return 'You are old enough to vote.'
+  }
+	else if (age >= 18 && age <=65 && !student) {
+		return 'You are eligible to vote.'
+  }
+  else {
+    return 'You are a senior citizen.'
+  }
+}
+
+console.log('Ex. 10:');
+
+console.log(checkVoteEligibility(17, true));
+console.log(checkVoteEligibility(18, true));
+console.log(checkVoteEligibility(22, false));
+console.log(checkVoteEligibility(70, false));
+
+// 11. Write a function that asks the user to enter their salary and their years of experience. If the salary is less than $30,000 and the years of experience are less than 5, print "Sorry, we cannot offer you the job at this time". If the salary is less than $50,000 and the years of experience are between 5 and 10 (inclusive), print "We can offer you the job at a lower salary". If the salary is greater than or equal to $50,000 or the years of experience are greater than or equal to 10, print "We can offer you the job at the requested salary".
+
+function decideTheSalaryOnExperience (salary, yearsOfExperience) {
+  if (salary < 30.000 && yearsOfExperience <5) {
```

Comment:
Numbers in JS don't include separators (unless you add decimals)
```suggestion
  if (salary < 30000 && yearsOfExperience <5) {
```